tutorial/create-test-virtual-device.md

```diff
@@ -0,0 +1,212 @@
+
+In this tutorial, we will go through the first steps of using Anbox Cloud to create a virtual Android device. By *virtual device*, we mean a simulated device that runs a plain Android operating system without any special apps installed. Anbox Cloud treats such a virtual device as an *empty* application, meaning an application that is not running a specific APK.
+
+By the end of this tutorial, we will be familiar with an {term}`Application` and an {term}`Instance` in Anbox Cloud that are used to create and stream the virtual Android device.
+
+This tutorial has two paths - you can use the CLI or the dashboard, depending on your learning goals. The CLI is more powerful and gives you access to all features of Anbox Cloud, while the dashboard is a simpler user interface. For this tutorial, it does not matter which path you choose.
+
+`````{tabs}
+````{group-tab} CLI
+
+We will use the {term}`Anbox Management Client (AMC)` in this tutorial to work with applications and instances. AMC communicates with the {term}`Anbox Management Service (AMS)`, the management module of Anbox Cloud.
+
+## Create the device
+
+The first step is to create a `yaml` file that can be used as the application manifest. It is important that this file is created in your `home` directory.
+
+In the home directory, let's first create a directory for the application content:
+
+      mkdir my-app
+
+This directory is the location where the application manifest file and any relevant APK must reside.
+
+For this tutorial, we are going to create a device with a basic application without an APK that will start directly with the Android system launcher. Let's create the `manifest.yaml` inside the `my-app` directory with the following contents:
+
+```yaml
+name: my-first-app
+resources:
+   cpus: 4
+   memory: 4GB
+   disk-size: 3GB
+```
+
+Now, let's create the application:
+
+      amc application create my-app/
+
+`my-app/` is the application directory where the manifest and any other application related artifacts are available.
+
+The application is created and bootstrapped. Now, to see the application details and its status, run:
+
+      amc application list
+
+Watch the output of this command till the application becomes *ready*.
+
+## Test the virtual device
+
+**Launch an instance**
+
+When the application is *ready*, let's launch an instance:
+
+      amc launch my-first-app --vm --enable-streaming
+
+This command creates and starts an instance for the application.
+
+`my-first-app` is the application name, you can also use the application ID to launch an instance for it.
+
+The `--enable-streaming` flag indicates that we want to stream this instance.
+
+The `--vm` flag indicates that we want to create the application inside a virtual machine instance instead of the default container instance.
```

Comment:
As an entry-level tutorial, we want users to experience launching an instance as quickly and simply as possible, so they can get a first impression of how Anbox Cloud instances operate. Launching a VM instance typically takes much longer than launching a container.
So maybe we should avoid using the VM instance in such a case.